CONTRIBUTING.md

```diff
@@ -161,9 +161,9 @@ The list is generally formatted like below.
 
 So you have the project page itself at the top. This can be the front-facing website or GitHub page.
 
-Next we ask to link to the contributing page. This will give direction to people who want to contribute. This file is important to read so you understand what is expected from contributors who have not contributed to their project before.
+Next, we ask to link to the contributing page. This will give direction to people who want to contribute. This file is important to read so that you understand what is expected from contributors who have not contributed to their project before.
 
-Last, we want to link to beginner friendly labels. These are typically `Good First Issue` or something similar.
+Lastly, we want to link to beginner-friendly labels. These are typical `Good First Issue` or something similar.
```

Comment:
Again, previous is more natural sounding if you read the code-formatted words as special labels rather than normal English you would read.

```suggestion
Lastly, we want to link to beginner-friendly labels. These are typically `Good First Issue` or something similar.
```

Reply:
Done.